R/get_data.R

```diff
@@ -144,6 +144,11 @@ bcdc_get_data.bcdc_record <- function(record, resource = NULL, ...) {
   cat("--------\n")
   cat("Please choose one option:")
   choices <- clean_wfs(resource_df$name)
+
+  ## To deal with situations where the resource names are the same
+  dup_choice <- length(choices) != length(unique(choices))
```

Comment:
Nitpicky, but simpler (and clearer intent) would be:

```suggestion
  dup_choice <- any(duplicated(choices))
```